components/create_post/create_post.jsx

```diff
@@ -9,7 +9,7 @@ import {Posts} from 'mattermost-redux/constants';
 
 import * as ChannelActions from 'actions/channel_actions.jsx';
 import * as GlobalActions from 'actions/global_actions.jsx';
-import {emitEmojiPosted} from 'actions/post_actions.jsx';
+import {createPost, emitEmojiPosted} from 'actions/post_actions.jsx';
```

Comment:
It would be better to turn this action into a redux-styled one and pass it in as a prop and I'd probably request changes on that if this was for master. Since it's for a release it's probably fine but can you make a ticket to change it over?

Reply:
Noted. I filed a ticket here [ICU-612](https://mattermost.atlassian.net/browse/ICU-612) and assigned to me.